sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/Statistics.scala

```diff
@@ -305,15 +260,15 @@ object ColumnStat extends Logging {
       percentiles: Option[ArrayData]): ColumnStat = {
     // The first 6 fields are basic column stats, the 7th is ndvs for histogram bins.
     val cs = ColumnStat(
-      distinctCount = BigInt(row.getLong(0)),
+      distinctCount = Option(BigInt(row.getLong(0))),
```

Comment:
nit: we should use `Some(value)` if `value` is expected to be not null.

Reply:
I'd keep it an Option, just to be prepared for more flexibility and more optionality, unless you have a strong opinion. (note: this code has moved to AnalyzeColumnCommand)